### DISCUSSION TITLE
Code scanning alerts: refs/pull/*/merge is not a branch

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#39167] [Code Security]
### Select Topic Area

Bug

### Body

https://github.com/check-spelling/krita/security/code-scanning/5299

[Code scanning alerts](https://github.com/check-spelling/krita/security/code-scanning) [#5299](https://github.com/check-spelling/krita/security/code-scanning/5299)

<img width="929" alt="image" src="https://user-images.githubusercontent.com/2119212/201962900-41da1905-22bc-4ce5-905b-88db501fcbd2.png">

Fixed in branch `refs/pull/2/merge` 18 minutes ago
@jsoref [Merge](https://github.com/check-spelling/krita/commit/5cb382da303cc1169c8eb2cadaa5c4db36560119) https://github.com/check-spelling/krita/commit/56a9e681884c8745998e0b725b53275c6f94ac01 [into](https://github.com/check-spelling/krita/commit/5cb382da303cc1169c8eb2cadaa5c4db36560119) https://github.com/check-spelling/krita/commit/7e5ab038c868d2e1e317894c84f821630c345d4a

---

The message should be "fixed in PR" or _something_.

In this case, the fix was actually made in the base branch...

**Comment by superstar1205** [comment]
Hi, I hope that you follow me. Thanks in Advance
